Birthday/Profile/ProfileScreen.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+
+struct ProfileScreen: View {
+  
+  @State private var isEditingModeOff = true
+  @State private var firstName = "Shirley"
+  @State private var lastName = "Peter"
+  
+  
+  private var fullName: String {
+    firstName + " " + lastName
+  }
+  //will be changed
+  private var areTextFieldsEdited: Bool {
+    firstName != "Shirley" || lastName != "Peter"
+  }
+  private var email = "ShirleyPeter01@gmail.com"
+
+  var body: some View {
+    ZStack {
+      Color(
+        red: 1,
+        green: 0.96,
+        blue: 0.96
+      )
+      .ignoresSafeArea()
+      VStack {
+        HStack {
+          Spacer()
+          //will be changed
+          Image(systemName: "envelope")
+            .resizable()
+            .frame(
+              width: 24,
+              height: 24
+            )
+        }
+        .padding(
+          EdgeInsets(
+            top: 0,
+            leading: 0,
+            bottom: 50,
+            trailing: 24
+          )
+        )
+        ProfileView(
+          isEditingModeOff: $isEditingModeOff,
+          firstName: $firstName,
+          lastName: $lastName,
+          fullName: fullName,
+          email: email
+        )
+        if isEditingModeOff {
+          Spacer()
+            .frame(height: 30)
+          Button(
+            "Edit Account"
+          ) {
+            isEditingModeOff.toggle()
+          }
+          .buttonStyle(PrimaryButtonStyle())
+          Button("Sign Out") {
+            //Logic for Sign out and back to Main Page
+          }
+          .buttonStyle(PrimaryButtonStyle())
+          Spacer()
+        } else {
+          Spacer()
```

Comment:
you can remove the spacers in lines 68 and 54, and add one spacer in line 53 before if